chat/cmd/main.go

```
@@ -0,0 +1,60 @@
+
+	pb "fitus-chat-service/proto"
+
+	"google.golang.org/grpc"
+)
+
+func main() {
+    // gRPC 서버 주소
+    serverAddr := "localhost:50051"
+
+    // gRPC 연결 설정
+    conn, err := grpc.Dial(serverAddr, grpc.WithInsecure())
```

golangci-lint (1.64.8)

SA1019: grpc.Dial is deprecated: use NewClient instead.  Will be supported throughout 1.x. 

(staticcheck)